### ISSUE TITLE
Separating out the user specific database connection

### ISSUE THREAD

**Issue by edward-burn** [mdlavallee92/CaprPhenotypePheburary#1]
Perhaps it would help to have a separate file for the database connection details which will be specific to each user? Maybe a CodeToRun file at the start where we set the connection details just the once, before creating the definitions for the different days?

**Comment by ablack3** [comment]
Yea that is a good idea. So each cohort creation R script would start by sourcing the codeToRun.R script to create the connection I gather. Each user would have their own version of codeToRun.R. @mdlavallee92 any thoughts on how best to handle different connection details?

**Comment by mdlavallee92** [comment]
yeah that would work. What is the standard for dealing with connection credentials in network studies?  If there is a standard we should use that, ow I like the codeToRun.R idea.

**Comment by edward-burn** [comment]
This is how I have it set up for one network study: https://github.com/oxford-pharmacoepi/CoagulopathyInVaccinated with the CodeToRun.R file being the only one that contains database specific information.

Possibly there is no "RunStudy" at the end though, and leave it to the user to work with specific definitions as they see fit

**Comment by mdlavallee92** [comment]
Yup looks good, I did something similar based on a different network study it should be in the extras folder now. I am going to change the .gitignore file so it ignores changes to the codeToRun too so you wont have to worry about that. Let me know if it needs anything else
